citest/google_session/google_session.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+
+from bs4 import BeautifulSoup
+
+class GoogleSession:
+  """Utility class for automating Google authentication.
+
+  Automates both the Google account login and scope approval to create an
```

Comment:
I dont really understand what this class is for. When I google "Google authentication" I am refered to "Google Authenticator" which looks like an android thing so dont think that's what you intend. Is this for OAuth? If so I'd create an OAuth support package and write the class you need as such, then perhaps a google wrapper around it that injects/configures the google endpoints.